src/main.js

```diff
@@ -35,18 +35,32 @@ app.get('/', (request, response) => {
 });
 
 app.get('/render/:url(*)', async(request, response) => {
-  const result = await renderer.serialize(request.params.url, request.query, config).catch((err) => console.error(err));
-  response.status(result.status).send(result.body);
+  try {
+    const result = await renderer.serialize(request.params.url, request.query, config);
+    response.status(result.status).send(result.body);
+  } catch (err) {
+    let message = `Cannot render ${request.params.url}`;
+    if (err && err.message)
+      message += ` - "${err.message}"`;
+    response.status(400).send(message);
```

Comment:
Unclear. But for the tested cases, eg. invalid URLs, 400 is more appropriate. The client requested a bad URL, it's not really an error on the server side.